switchlink/switchlink_db.h

```diff
@@ -36,9 +37,9 @@ typedef struct switchlink_db_tuntap_info_ {
   switchlink_handle_t tunp_h;
   switchlink_mac_addr_t mac_addr;
   switchlink_link_type_t link_type;
-  //struct tuntap_flags {
-    //bool <?>_enabled;
-    //uint8_t <?>_mode;
```

Comment:
(a) You're quite right, but (b) *not in this PR*.

Sweeping changes of the sort this CL makes should be separate from other kinds of changes. We're relying on the integrity of a tool or a process (rather than the acuity of the reviewers) for the correctness of the changes. We're talking about 120 files here!

With maybe one BSR edit that slipped through inadvertently, the only non-mechanical changes in this CL are the ones needed to address issues that surfaced when clang-format reordered some of the #includes.

That's why I separated the CL itself into three commits.

To quote from Google's [Standard of Code Review](https://google.github.io/eng-practices/review/reviewer/standard.html):

> In general, reviewers should favor approving a CL once it is in a state where it definitely improves the overall code health of the system being worked on, even if the CL isn’t perfect.